test/csv/test_data_converters.rb

```diff
@@ -103,4 +103,89 @@ def test_builtin_date_time_converter_iso8601_utc
     assert_equal(datetime,
                  CSV::Converters[:date_time][iso8601_string])
   end
+
+  def test_builtin_date_time_converter_rfc3339_minute
+    iso8601_string = "2018-01-14 22:25"
```

Comment:
Could you use `rfc3339_string` instead of `iso8601_string` because this is not ISO 8601 format?

Reply:
done.